Say Their Names/Source/Decoder/PeopleDecoder.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PersonSerialization.swift
+//  Say Their Names
+//
+//  Created by evilpenguin on 6/3/20.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import Foundation
+
+final class PeopleDecoder {
+    public class func decode(_ data: Any?) -> People? {
```

Comment:
Already working on it.

Reply:
> Since it looks like the decoded objects are decoding to optional values, does it make sense to have two different types of objects (i.e. `WirePerson` & `Person`)? This would allow us to safely decode the objects coming from the network and have them mapped over to an object with non-optional values.

Agreed. It will not be optional as well once I add the db layer.

Reply:
> > Since it looks like the decoded objects are decoding to optional values, does it make sense to have two different types of objects (i.e. `WirePerson` & `Person`)? This would allow us to safely decode the objects coming from the network and have them mapped over to an object with non-optional values.
> 
> Agreed. It will not be optional once I add the db layer.

Pushed